src/content/docs/data-apis/manage-data/drop-data-using-nerdgraph.mdx

```diff
@@ -56,7 +56,7 @@ The following data types can be targeted for data dropping:
 * [Default infrastructure monitoring events](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data) and [infrastructure integrations](/docs/integrations/infrastructure-integrations/get-started/introduction-infrastructure-integrations) events. Some caveats:
   * When you drop this data, the raw data is dropped, but the aggregated `SystemSample`, `ProcessSample`, `NetworkSample` and `StorageSample` events are still available (for more on this, see [Data retention](/docs/data-apis/manage-data/manage-data-retention/#infrastructure-data)). Though still available, this data doesn't count towards ingest and is not billable.
   * Raw infrastructure data is used for alerting, so if you drop that data, you can't alert on it. Because the aggregated data is still available, you may still see that data in charts with time ranges above 59 minutes.
-* [Dimensional metrics](/docs/telemetry-data-platform/ingest-manage-data/understand-data/new-relic-data-types#dimensional-metrics) (the `Metric` data type). Some caveats:
+* [Dimensional metrics](/docs/telemetry-data-platform/ingest-manage-data/understand-data/new-relic-data-types#dimensional-metrics) (the `Metric` data type, _not_ including metrics with a `newrelic.timeslice.value` or with a name beginning with `apm.*`). Some caveats:
```

Comment:
Similar to my last comment, I think it's better to just plainly state that metric timeslice data cannot be dropped, and then link to the docs on querying timeslice data for further details on how to identify/query that particular data type.

Your mileage on this may vary, but my suggestions here are:

* Remove the statement in parentheses (including the preexisting, overly broad statement about "the Metric data type")
* Add another bullet point under "Some caveats" that states "Metric timeslice data cannot be dropped," while linking out to the doc on querying timeslice data with NRQL

Reply:
I do think it's important to have some kind of info which tells people what metric timeslice data is though and I found the doc I linked more helpful for understanding the difference between the two types of metrics. It uses the same doc which explains what Dimensional metrics are so to me it made it more clear what these are and how they are different. Which I think is something few of our users understand but do need to know.

I did remove the entire `()` section but left the link in place. I did _also_ link your recommended doc in the other section about timeslice metrics. I'm open to more disagreeing feedback though.